src/post/body.js

```diff
@@ -1,32 +1,53 @@
-let React = require('react'),
-  _ = require('lodash'),
-  steemembed = require('steemembed'),
-  striptags = require('striptags'),
-  marked = require('marked');
+import React from 'react';
+import { has } from 'lodash';
+import steemembed from 'steemembed';
+import striptags from 'striptags';
+import marked from 'marked';
 
-module.exports = React.createClass({
-  render() {
-    const embeds = steemembed.getAll(this.props.body);
-    let body = this.props.body;
-    body = striptags(marked(body), ['a', 'p', 'h1', 'h2', 'h3', 'img', 'code', 'pre']);
-    let jsonMetadata = {};
-    try { jsonMetadata = JSON.parse(this.props.jsonMetadata); }
-    catch (e) { }
-    if (_.has(jsonMetadata, 'image[0]')) {
+marked.setOptions({
+  renderer: new marked.Renderer(),
+  sanitize: true,
+  smartypants: true
+});
+
+export default (props) => {
+  const embeds = steemembed.getAll(props.body);
+  let body = props.body;
+  // body = striptags(remarkable.render(body), ['a', 'p', 'h1', 'h2', 'h3', 'img', 'code', 'pre']);
```

Comment:
If a code is not necessary always remove rather than commenting. If we need it later we can always use git history for that.